src/main/app/Application.cpp

```diff
@@ -97,11 +97,13 @@ void Application::init() {
 	IPausedScreenShPtr pausedScreen(new PausedScreen(styleManager));
 	ILoadingScreenShPtr loadingScreen(new LoadingScreen(styleManager));
 	IIntroScreenShPtr introScreen(new IntroScreen(styleManager, eventManager));
+	IGameOverScreenShPtr gameOverScreen(new GameOverScreen(styleManager, eventManager));
 	ICreditsScreenShPtr creditsScreen(new CreditsScreen(styleManager));
 	view->addScreen(menuScreen);
 	view->addScreen(pausedScreen);
 	view->addScreen(loadingScreen);
 	view->addScreen(introScreen);
+	view->addScreen(gameOverScreen);
 	view->addScreen(creditsScreen);
 
```

Comment:
Probably add this to a refactor task list, to sort this list into pairs in a logic order. If we have  a future game with 30 screens this could get annoying to read.